lib/datadog/tracing/contrib/sidekiq/server_tracer.rb

```diff
@@ -61,7 +61,11 @@ def call(worker, job, queue)
               span.set_tag(Ext::TAG_JOB_RETRY_COUNT, job['retry_count'])
               span.set_tag(Ext::TAG_JOB_QUEUE, job['queue'])
               span.set_tag(Ext::TAG_JOB_WRAPPER, job['class']) if job['wrapped']
-              span.set_tag(Ext::TAG_JOB_DELAY, 1000.0 * (Core::Utils::Time.now.utc.to_f - job['enqueued_at'].to_f))
+
+              enqueued_at = job['enqueued_at']
+              # Sidekiq 8.0+ uses Integer milliseconds for enqueued_at; prior versions use Float seconds
+              enqueued_at /= 1000.0 if enqueued_at.is_a?(Integer)
```

Comment:
I would like to ask for a constant extraction with a name that will reflect what comment tries to explain

Reply:
I've added `Ext::SIDEKIQ_8_SECONDS_PER_INTEGER` and changed this to `*=`.